releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -0,0 +1,20 @@
+# azure-appconfiguration-provider
+
+[Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
+
+## 1.0.0b1 - October 13, 2022
+
+Initial Beta Release of the Azure App Configuration Provider for Python
+
+### New Features
+
+* Connecting to an App Configuration Store using a connection string or Azure Active Directory.
+* Selecting multiple sets of configurations using SettingSelector.
+* Trim prefixes off key names.
+* Resolving Key Vault References, requires AAD.
+* Secret Resolver, resolve Key Vault References locally without connecting to Key Vault.
+* Json Content Type
```

Comment:
> Json Content Type

What does this mean?

Reply:
Not sure of the correct way to word this. We support `application/json` and similar and return a json object.